### PR TITLE
Add sponsor links to project

### PR DESCRIPTION
**Pull request by biohazard999**
- Adds link to GH sponsors
- Adds link to Patreon page
- Adds link with redirect to blog
- Removes shout out to console in overload tests

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,21 +1,57 @@
...
+
+In order to receive this license, you have to agree to its rules.  Those rules are both obligations under that agreement and conditions to your license.  Don't do anything with this software that triggers a rule you can't or won't follow.
+
+## Notices
+
+Make sure everyone who gets a copy of any part of this software from you, with or without changes, also gets the text of this license and the contributor and source code lines above.
+
+## Commercial Trial
+
+Limit your use of this software for commercial purposes to a thirty-day trial period.  If you use this software for work, your company gets one trial period for all personnel, not one trial per person.
```
Comment (hazard999) [inline LICENSE:21]: Doesn't match with [https://github.com/xenial-io/Tasty/pull/81/files#diff-04c6e90faac2675aa89e2176d2eec7d8R202-R203](https://github.com/xenial-io/Tasty/pull/81/files#diff-04c6e90faac2675aa89e2176d2eec7d8R202-R203)

Comment (hazard999) [reply LICENSE:21]: thirty-day trial period vs a 32 day trial period

File: docs/input/support/index.cshtml [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+        <div class="text-center my-3">
+            <i class="fad fa-heart fa-5x icon-red"></i>
+        </div>
+        <p>
+            Tasty isn't developed by a big, faceless corporation or backed by mountains of venture captital.<br>
+            It's mostly <a href="https://github.com/biohazard999" target="_blank">Manuel</a> and other awesome contributors spending lots of nights and weekends as a labor of love.
+        </p>
+        <p>
+            Development and support of Tasty has been taken of real quick and there are no plans to slow down.<br>
```
Comment (Paramethod) [inline docs/input/support/index.cshtml:18]: Rephrase. You mean 'take of' like ✈?
Perhaps:
Development and support of Tasty took off quickly and there are no plans to slow down.

File: Directory.Build.props [hunk 3]
```diff
@@ -17,7 +16,12 @@
     <IncludeSource>True</IncludeSource>
     <IncludeSymbols>true</IncludeSymbols>
     <SymbolPackageFormat>snupkg</SymbolPackageFormat>
-    <PackageRequireLicenseAcceptance>False</PackageRequireLicenseAcceptance>
+    <PackageProjectUrl>https://tasty.xenial.io</PackageProjectUrl>
+    <RepositoryUrl>https://github.com/xenial-io/Tasty</RepositoryUrl>
+    <RepositoryType>git</RepositoryType>
+    <PublishRepositoryUrl>true</PublishRepositoryUrl>
+    <PackageLicenseFile>LICENSE</PackageLicenseFile>
```
Comment (Paramethod) [inline Directory.Build.props:23]: LICENSE file should have .md extension.
https://docs.microsoft.com/en-us/nuget/reference/nuspec#license

Comment (biohazard999) [reply Directory.Build.props:23]: They can but not have to. Just checked, license acceptance works, and it will be linked as text which is fine I guess.

Comment (biohazard999) [reply Directory.Build.props:23]:
```xml
    <requireLicenseAcceptance>true</requireLicenseAcceptance>
    <license type="file">LICENSE</license>
```

Comment (Paramethod) [reply Directory.Build.props:23]: My editor would thank you for letting him know it is markdown content 😉

File: LICENSE (hunk 1 above)
Comment (biohazard999) [reply LICENSE:21]: Good catch! Fixed.

File: docs/input/support/index.cshtml (hunk 2 above)
Comment (biohazard999) [reply docs/input/support/index.cshtml]: Fixed

File: Directory.Build.props (hunk 3 above)
Comment (biohazard999) [reply Directory.Build.props]: Okay good point. (I use VSCode for most of the metadata stuff, so I didn't mind so much) should be fixed

**Comment by biohazard999** [comment]
Looks fine to me! :shipit: ?

**Comment by Paramethod** [comment]
LGTM :shipit:

**Pull request merged**